test/functional/api/v1/dependencies_controller_test.rb

```diff
@@ -31,6 +31,23 @@ class Api::V1::DependenciesControllerTest < ActionController::TestCase
     end
   end
 
+  # INVALID GEMS:
+  context "On GET to index --> with invalid gems --> JSON" do
+    setup do
+      rubygem = create(:rubygem, name: "testgem")
+      @version = create(:version, number: "1.0.0", rubygem_id: rubygem.id)
```

Comment:
do we need the above two lines in the setup?

Reply:
probably not, I just copied tests around doing the same (even not used)